python/torch_mlir/extras/onnx_importer.py

```diff
@@ -603,6 +723,13 @@ def type_proto_to_type(self, tp: onnx.TypeProto) -> IrType:
             element_type = self.get_optional_element_type(ot.elem_type)
             return self.get_optional_type(element_type)
 
+        # Check if TypeProto is empty (sometimes happens for unused function
+        # arguments)
+        if tp.SerializeToString(
+            deterministic=True
+        ) == onnx.TypeProto().SerializeToString(deterministic=True):
```

Comment:
This one is awkward but that might just be Protobuf's fault. Alternatively maybe I've missed something. It really feels like there should be a better way.

Reply:
`onnx.TypeProto().WhichOneof("value") is None`

https://protobuf.dev/reference/python/python-generated/

Reply:
Aha, thanks for the pointer. I'd only looked at the generated documentation for Protobuf's classes so far. I'll try this out.